src/elements/content-sidebar/Sidebar.js

```diff
@@ -103,8 +107,40 @@ class Sidebar extends React.Component<Props, State> {
             this.setForcedByLocation();
             this.setState({ isDirty: true });
         }
+
+        // Active annotation id changed. If location is currently an annotation path or
+        // if location is not currently an annotation path but the active annotation id
+        // transitioned from falsy to truthy, update the location accordingly
+        if (
+            prevActiveAnnotationId !== activeAnnotationId &&
+            (isAnnotationsPath || (activeAnnotationId && !isAnnotationsPath))
+        ) {
+            this.updateActiveAnnotation();
+        }
     }
 
+    updateActiveAnnotation = () => {
+        const {
+            annotatorState: { activeAnnotationId } = {},
```

Comment:
We default the value here, but not in componentDidUpdate. Should this be a `defaultProp` value?